.github/workflows/test-monai-deploy-express.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Test MONAI Deploy Express
+on:
+  push:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+# Controls when the workflow will run
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      
+      - name: Start MONAI Deploy Express
+        working-directory: ./deploy/monai-deploy-express
+        run: docker compose up -d --wait
+        
+      - name: Informatics Gateway Health Check
+        continue-on-error: true
+        run: |
+          curl http://localhost:5000/health
+
+      - name: Workflow Manager Health Check
+        continue-on-error: true
+        run: |
+          curl http://localhost:5001/health
+
+      - name: Task Manager Health Check
+        continue-on-error: true
+        run: |
+          curl http://localhost:5002/health
+        
```

Comment:
Might be worth adding a docker compose down in there too. I know it won't make any difference to the cleaning up process, but it might flag another issue. If added then we'd need another test to make sure all docker containers had been cleaned up.

Reply:
Added new step to tear down